addons/mail/static/tests/helpers/widget_extractor.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module **/
```

Comment:
Maybe move in a legacy subfolder?

Also I probably asked already but why do we still need widget?